ci/scripts/run-vulnerability-scan/create-jira-issue.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/ash
+# shellcheck shell=dash
+set -euo pipefail
+
+FILE_NAME=$(ls reports)
+
+echo "Vulnerability scan report file name: $FILE_NAME .."
+
+response=$(curl --request POST \
```

Comment:
Can you add `--fail` to this curl, otherwise it will carry on even if the curl to jira fails)

Reply:
👍🏼 Updated!